docs/docs/frontend-protocol.md

```diff
@@ -241,6 +241,74 @@ Sends a custom rpc command to the named receiver. This may be a notification
 or a request.
 
 
+### Find and replace methods
+
+#### find
+
+`find {"chars": "a", "case_sensitive": false, "regex": false, "whole_words": true}`
+Parameters `regex` and `whole_words` are optional and by default `false`.
+
+Sets the current search query and options.
+
+#### find_next and find_previous
+
+`find_next {"wrap_around": true, "allow_same": false, "modify_selection": "set"}`
+`find_previous {"wrap_around": true, "allow_same": false, "modify_selection": "set"}`
```

Comment:
I think at least `allow_same` should get a bit of additional documentation, since I don't think it's usage is always clear.

Reply:
Done :)